beacon-chain/core/blocks/signature.go

```diff
@@ -74,6 +74,21 @@ func VerifyBlockSignature(beaconState *stateTrie.BeaconState, block *ethpb.Signe
 	return helpers.VerifyBlockSigningRoot(block.Block, proposerPubKey, block.Signature, domain)
 }
 
+// VerifyBlockHeaderSignature verifies the proposer signature of a beacon block header.
+func VerifyBlockHeaderSignature(beaconState *stateTrie.BeaconState, header *ethpb.SignedBeaconBlockHeader) error {
```

Comment:
We should add a test for this to make codecov happy

Reply:
All functions in this file are untested 😮. We should likely amend in a separate PR. I will add a test for this particular one in this PR to keep diff simple

Reply:
Added